File: JAVA/Daily Flash Codes/Week1/Day5_Jan17/21August/prog1.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int radius;	//radius
		
		System.out.println("Enter the radius: ");
		radius = Integer.parseInt(in.readLine()) ;		//take radius from user

		double area = 3.14 * radius *radius;

		System.out.println("Area of Circle: "+ area);
	
	}
}

File: C/Daily Flash/Week1/Day5_Jan17/code_C/prog1.c
/*Program 1: Write a Program to calculate area of circle.
 * */

#include <stdio.h>

#define PI 3.14
void main() {
	
	int radius;
	printf("Enter the radius of circle: ");	
	scanf( "%d" , &radius);		//take radius

	float area = PI * radius *radius;

	printf("Area of circle: %f\n",area);
}
